src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,32 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { user, body, title, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        {user && (
+          <p>
+            {' Posted by  '}
+            <UserInfo user={user} />
```

Comment:
<img width="554" alt="image" src="https://github.com/user-attachments/assets/00983816-6da0-4d08-ac39-7df137dde817">
the find method can returns the undefined, so it's better to add a check
```suggestion
            {user && <UserInfo user={user} />}
```